modules/vaos/app/services/vaos/appointment_requests_service.rb

```diff
@@ -31,6 +31,19 @@ def get_requests(start_date = nil, end_date = nil)
       end
     end
 
+    def put_cancel_request(request_object_body)
+      params = VAOS::Patient.new(request_object_body).params
+      params.merge!(patient_identifier: { unique_id: user.icn, assigning_authority: 'ICN' })
+
+      binding.pry
```

Comment:
these binding.pry are here as breakpoints to help facilitate recording cassettes. They will be removed.